### ISSUE TITLE
Punctuation characters not announced when used in aria-keyshortcuts

### ISSUE THREAD

**Issue by stoyanovski** [nvaccess/nvda#13924]
### Steps to reproduce:

> The issue is reproducible in the following example: https://jsbin.com/cizayaheyi/edit?html,output
> Start NVDA and set the focus on the button

### Actual behavior:

> NVDA announces: "Control plus"

### Expected behavior:

> The keyboard shortcut should be announced as "Control plus comma"

#### Other information:

> WAI-ARIA Specification for aria-keyshortcuts: https://rawgit.com/w3c/aria/master/#aria-keyshortcuts
> Examples of valid keyboard shortcuts include:
> - "A"
> - "Shift+Space"
> - "Control+Alt+." (the dot is not announced either)
> - "Control+Shift+&#39;"
> - "Alt+Shift+P Control+F"
> - "Meta+C Meta+Shift+C"

**Comment by feerrenrut** [comment]
I can reproduce this. It seems the punctuation level affects the reporting of the shortcut. At "some" the comma isn't reported, at 'all' it is. This is a bug in NVDA.
